css/css-masking/clip-path/reference/clip-path-shape-004-ref.html

```diff
@@ -0,0 +1,24 @@
+<!DOCTYPE html>
+<html>
+<head>
+  <title>CSS Masking: Test clip-path property and shape function with nonzero fill</title>
+  <link rel="help" href="https://drafts.csswg.org/css-shapes-2/#funcdef-shape">
+  <meta name="assert" content="The clip-path property takes the basic shape
+	'shape()' for clipping. Test arcs.">
```

Comment:
nit: We don't need to add meta "assert" for reference file

Reply:
Fixed